src/components/CodeView/index.tsx

```diff
@@ -70,6 +70,11 @@ export class CodeViewBase extends React.Component<Props> {
 
     return (
       <React.Fragment>
+        {selectedMessageMap &&
+          selectedMessageMap.global.length > 0 &&
+          isLineSelected(getCodeLineAnchorID(0), location) && (
+            <div id={getCodeLineAnchorID(0)} ref={_scrollToSelectedLine} />
+          )}
         {selectedMessageMap &&
           selectedMessageMap.global.map((message) => {
             return <LinterMessage key={message.uid} message={message} />;
```

Comment:
Is there an issue on file about giving linter messages a selected style? I know we talked about it. If not, please file it. I think it will help to show what's going on when there are multiple messages.

Reply:
I filed #826